src/Illuminate/Foundation/Console/RouteListCommand.php

```diff
@@ -116,7 +116,11 @@ protected function getRoutes()
             return $this->getRouteInformation($route);
         })->filter()->all();
 
-        $routes = $this->sortRoutes($this->option('sort'), $routes);
+        if (($sort = $this->option('sort')) !== null) {
+            $routes = $this->sortRoutes($sort, $routes);
+        } else {
+            $routes = $this->sortRoutes('uri', $routes);
+        }
```

Comment:
The option is by 'uri' by default, so it seems not needed, as it's never null. I added a test to be sure of it.